sklearn/feature_extraction/text.py

```diff
@@ -607,9 +607,10 @@ class CountVectorizer(BaseEstimator, VectorizerMixin):
         A mapping of terms to feature indices.
 
     stop_words_ : set
-        Terms that were ignored because
-        they occurred in either too many
-        (`max_df`) or in too few (`min_df`) documents.
+        Terms that were ignored because they either
+        - occurred in too many documents (`max_df`)
+        - occurred in too few documents (`min_df`)
+        - were cut off by feature selection (`max_features`).
```

Comment:
You need to make sure that the development version of scikit-learn is the one used when building the doc. To do so, you can run `python setup.py develop` in your source folder. Once the doc is build you can uninstall and reinstall the latest stable version of scikit-learn if you prefer not to use the development version otherwise.